torch/quantization/observer.py

```diff
@@ -438,7 +603,7 @@ def _compute_quantization_error(self, next_start_bin, next_end_bin, norm_type):
         return norm
 
     def _non_linear_param_search(self):
-        """
+        r"""
```

Comment:
We should copy the comments from norm_minimization.cc here so that it is visible to end users

Reply:
I added the algo description in the class docstring